### PR TITLE
Feature/657 remove cmdstan guide

### PR DESCRIPTION
**Pull request by mitzimorris**
#### Submisison Checklist

- [ ] Run tests: `./runCmdStanTests.py src/test`
- [x] Declare copyright holder and open-source license: see below

#### Summary:

CmdStan manual docs are now online, remove old latex versions and makefile tasks.

Also add logic to `make/command` to address issue #904 

#### Intended Effect:

Repo contains source code only, all docs are under repo stan-dev/docs

#### How to Verify:

N/A

#### Side Effects:

need to modify CI and Jenkins config; also release scripts

#### Documentation:

N/A

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

**Comment by mitzimorris** [comment]
@serban-nicusor-toptal - can you help me with the CI config - we no longer need to build the manual.

we will also need to change the release scripts - pdf for CmdStan guide will be built as part of docs repo.

**Comment by serban-nicusor-toptal** [comment]
Hey @mitzimorris I've created a PR in ci-scripts for the cmdstan release script to not build the manual anymore.
Related to the CI part, remove the manual building from [here](https://github.com/stan-dev/docs/blob/master/Jenkinsfile#L136) ?

**Comment by mitzimorris** [comment]
> Related to the CI part, remove the manual building from here ?
(https://github.com/stan-dev/docs/blob/master/Jenkinsfile)
yes.

that script will need to be modified at https://github.com/stan-dev/docs/blob/109727498206499a0ef24fd8f037655836045129/Jenkinsfile#L49

on the docs repo side, there's more work on the python scripts to build and add redirects
for the cmdstan-guide section, and I'll update the documentation pages on the website
(stan-dev.github.io)

File: make/command [hunk 1]
```diff
@@ -18,14 +22,17 @@ else
...
+#endif
 
 .PRECIOUS: bin/print$(EXE) bin/stansummary$(EXE) bin/diagnose$(EXE)
 bin/print$(EXE) bin/stansummary$(EXE) bin/diagnose$(EXE) : CPPFLAGS_MPI =
 bin/print$(EXE) bin/stansummary$(EXE) bin/diagnose$(EXE) : LDFLAGS_MPI =
 bin/print$(EXE) bin/stansummary$(EXE) bin/diagnose$(EXE) : LDLIBS_MPI =
 bin/print$(EXE) bin/stansummary$(EXE) bin/diagnose$(EXE) : bin/%$(EXE) : bin/cmdstan/%.o $(BOOST_PROGRAM_OPTIONS_LIB)
 	@mkdir -p $(dir $@)
-	$(LINK.cpp) $^ $(BOOST_PROGRAM_OPTIONS_LIB) $(LDLIBS) $(OUTPUT_OPTION)
+	$(LINK.cpp) -fvisibility=hidden -fvisibility-inlines-hidden $^ $(BOOST_PROGRAM_OPTIONS_LIB) $(LDLIBS) $(OUTPUT_OPTION)
```
Comment (SteveBronder) [inline make/command:35]: Why did these go from only Darwin to always on? Are these just optims or is there a reason we need symbols to be hidden here?

**Comment by mitzimorris** [comment]
good catch.  I was working in the wrong branch.
will correct.

File: make/command (hunk 1 above)
Comment (mitzimorris) [reply make/command:35]: will change - that isn't correct and shouldn't have gotten mixed up with this PR.
fixed the makefile problem as part of PR #907

**Comment by mitzimorris** [comment]
` src/test/interface/example_output/bernoulli_chain_1.csv` was added as part of #907, so it's there now.

**Pull request merged**